src/Parallel/Tags/InputSource.hpp

```diff
@@ -0,0 +1,36 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <string>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "Options/Options.hpp"
+
+namespace Parallel::OptionTags {
+/// \ingroup OptionTagsGroup
+/// An input file tag tag specifying the yaml input-file source itself,
+/// so that the yaml input-file string can be embedded in H5 output.
+struct InputSource {
```

Comment:
why does this tag exist?  can't you use the `Options::Tags::InputSource` ?

Reply:
Yes, you can. I was confused from a slack conversation we had about this, which made me incorrectly think I needed another option tag. After the fixups, I just use Options::Tags::InputSource